src/decisionengine/framework/util/fs.py

```diff
@@ -27,10 +30,10 @@ def files_with_extensions(dir_path, *extensions):
                 channel_name = os.path.splitext(entry.name)[0]
                 name_to_path.append([channel_name, str(entry)])
     except FileNotFoundError:
-        logging.getLogger("decision_engine").exception("invalid path to config file given")
+        logger.exception("invalid path to config file given")
         raise
-    except Exception:  # pragma: no cover
-        logging.getLogger("decision_engine").exception("Unexpected error!")
+    except Exception:
```

Comment:
Any chance to get `# pragma: no cover` put back so the test coverage knows this isn't something we can test?

Reply:
Done.